.github/workflows/push-container.yaml

```diff
@@ -23,13 +23,16 @@ jobs:
               with:
                 node-version: '18.x'
             - uses: docker/login-action@v3
+              if: github.repository == 'nangohq/nango'
               with:
                 username: ${{ secrets.DOCKER_USERNAME }}
                 password: ${{ secrets.DOCKER_PASSWORD }}
             - name: Build and push container
+              env:
+                PUSH: ${{ (github.repository == 'nangohq/nango') && '--push' || '' }}
```

Comment:
`--push` is short for `--output type=registry` 
https://docs.docker.com/engine/reference/commandline/buildx_build/#docker